.changeset/yellow-goats-travel.md

```diff
@@ -0,0 +1,79 @@
+---
+'@openfn/language-fhir': major
+---
+
+### Added
+
+- Improved type definitions for `request()` and `get()`
+
+### Changed
+
+- Updated `request` function to the new signature:
+
+  ```js
+  export const request =
+    async (method, path, options = {}, callback) =>
+    state => ({
+      ...state,
+      data,
+      response,
+    });
+  ```
+
+- Updated `post` function to the new signature:
+
+  ```js
+  export const post = (path, data, options, callback) => state => ({
+    ...state,
+    data,
+    response,
+  });
+  ```
+
+- Updated `get` function to the new signature:
+
+  ```js
+  export const get = (path, params, options, callback) => state => ({
+    ...state,
+    data,
+    response,
+  });
+  ```
+
+- Updated `create` function to the new signature:
+
+  ```js
+  export const create = (resource, resource, params, callback) => state => ({
+    ...state,
+    data,
+    response,
+  });
+  ```
+
+- Updated `createTransactionBundle` function to the new signature:
+
+  ```js
+  export const createTransactionBundle =
+    (entries, params, callback) => state => ({
```

Comment:
We'll have to fix the signature here as well

Reply:
Done ✅ ,  i have update the signature